packages/services/src/view-service/fees.test.ts

```diff
@@ -21,8 +20,8 @@ describe('extractAltFee', () => {
         }),
       ],
     });
-    const result = extractAltFee(request);
-    expect(result.equals(umAssetId)).toBeTruthy();
+    const outputAsset = request.outputs.map(o => o.value?.assetId).find(Boolean);
+    expect(outputAsset!.equals(umAssetId)).toBeTruthy();
```

Comment:
🚨 This (and the three changed tests below) are no longer testing `extractAltFee()`, since they don't call it. They should be changed to actually call `extractAltFee()`, or else they're essentially useless tests.

Reply:
we're passing in an indexDB now, not sure how we'd simulate this in the test since MockIndexDB has different interface?

Reply:
Un-resolving this because this is a pretty important issue: the tests here are literally not testing anything. It's the equivalent of writing an `add()` function, then writing a test like this:

```ts
it('adds', () => {
  const result = 2 + 2;
  expect(result).toBe(4);
})
```

In this test, we haven't called the `add()` function, so if we break the `add()` function later, the tests won't catch it.

Happy to pair on updating `MockedIndexedDB` if that would help!

Reply:
Talked offline. Given this blocks deployment, we want to move forward. However, we should absolutely follow up with this with a separate PR.